### PR TITLE
Fix `make lint` failures

### PR DESCRIPTION
**Pull request by abhinav**

### CODE REVIEW

File: encoding/x/protobuf/inbound.go [hunk 1]
```diff
@@ -81,13 +81,11 @@ func (u *unaryHandler) Handle(ctx context.Context, transportRequest *transport.R
 	var wireError *wirepb.Error
 	if appErr != nil {
 		responseWriter.SetApplicationError()
-		wireError = &wirepb.Error{
-			appErr.Error(),
-		}
+		wireError = &wirepb.Error{Message: appErr.Error()}
```
Comment (peter-edge) [inline encoding/x/protobuf/inbound.go]: I dont want to do composite keys on purpose to verify at compile time if the message is completely filled out appropriately @sectioneight

File: encoding/x/protobuf/inbound.go [hunk 2]
```diff
@@ -81,13 +81,11 @@ func (u *unaryHandler) Handle(ctx context.Context, transportRequest *transport.R
...
 	}
 	wireResponse := &wirepb.Response{
-		responseData,
-		wireError,
+		Payload: responseData,
+		Error:   wireError,
```
Comment (peter-edge) [inline encoding/x/protobuf/inbound.go]: Same comment

**Comment by peter-edge** [comment]
None of this should fail on lint, can you show me the errors? This has been passing locally and on Travis CI, and I do not want this change to be made.

**Comment by abhinav** [comment]
```

govet failed:
encoding/x/protobuf/inbound.go:84: go.uber.org/yarpc/encoding/x/protobuf/internal/wirepb.Error composite literal uses unkeyed fields
encoding/x/protobuf/inbound.go:88: go.uber.org/yarpc/encoding/x/protobuf/internal/wirepb.Response composite literal uses unkeyed fields
encoding/x/protobuf/testing/testing_test.go:99: go.uber.org/yarpc/internal/examples/protobuf/examplepb.GetValueRequest composite literal uses unkeyed fields
encoding/x/protobuf/testing/testing_test.go:109: go.uber.org/yarpc/internal/examples/protobuf/examplepb.SetValueRequest composite literal uses unkeyed fields
encoding/x/protobuf/testing/testing_test.go:116: go.uber.org/yarpc/internal/examples/protobuf/examplepb.FireRequest composite literal uses unkeyed fields
internal/examples/protobuf/main.go:83: go.uber.org/yarpc/internal/examples/protobuf/examplepb.GetValueRequest composite literal uses unkeyed fields
internal/examples/protobuf/main.go:101: go.uber.org/yarpc/internal/examples/protobuf/examplepb.SetValueRequest composite literal uses unkeyed fields
internal/examples/protobuf/main.go:113: go.uber.org/yarpc/internal/examples/protobuf/examplepb.FireRequest composite literal uses unkeyed fields
internal/examples/protobuf/example/example.go:89: go.uber.org/yarpc/internal/examples/protobuf/examplepb.GetValueResponse composite literal uses unkeyed fields
```

**Comment by peter-edge** [comment]
What version of go are you on?

**Comment by peter-edge** [comment]
Ie note it works on go 1.8 on Travis :) This is another reason why I need
to get the Docker thing done.

**Comment by abhinav** [comment]
Ah, I'm on 1.7.

**Comment by peter-edge** [comment]
Ya, go vet changes between versions a bit heh.

image

**Comment by akshayjshah** [comment]
As long as vet passes in CI, I'm okay with dropping this.

**Comment by abhinav** [comment]
upgraded local Go

**Pull request closed without merging**